test/remote/gateways/remote_mondido_test.rb

```diff
@@ -0,0 +1,1487 @@
+require 'test_helper'
+
+class RemoteMondidoTest < Test::Unit::TestCase
```

Comment:
Overall this remote test looks way over-complicated; you shouldn't need all the support methods. Reference other gateway remote tests for how to use methods on the gateway (such as `store`) instead of re-implementing in the test.